CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## main / unreleased
 
-### Grafana Mimir
+### Grafana Mimit r
```

Comment:
```suggestion
### Grafana Mimir
```

Reply:
My keyboard might jump around. Will be more careful next time.